### DISCUSSION TITLE
Left clicking on nm-applet  in LxQt panel does not work

### DISCUSSION THREAD

**Discussion by Lokawn** [lxqt/lxqt#2253] [Q&A] [answered]
Left clicking on nm-applet in LxQt panel does not work, only right clicking works. I think it should be opposite like on KDE.
Same thing happens with the blueman applet.
I tried the fedora LxQt spin and nm-applet works as desired there.
I am using Arch Linux

https://user-images.githubusercontent.com/39772448/174251352-ed4df88c-df28-41e1-b847-eb47d494f93c.mp4

.

**Comment by tsujan** [comment]
Not reproducible here:

https://user-images.githubusercontent.com/981076/174253345-3c274e38-b662-4275-81f8-31363e5d4d22.mp4

Ignore the menu offset; it's a known bug in nm-applet. I use Manjaro, which isn't different from Arch in this regard.

**Reply by tsujan** [reply]
Oh, sorry, the above video was for right clicking. This one is for left clicking:

https://user-images.githubusercontent.com/981076/174255186-00cb2b13-f8c7-46e4-bb4d-0febfe97d75b.mp4

I use both on a regular basis.

**Reply by Lokawn** [reply]
In my install both left and right click menu are combined in a single menu, which is only visible on right click

https://user-images.githubusercontent.com/39772448/174257139-b281d1f5-8608-422f-b692-3d70f74ad661.mp4

.

**Reply by tsujan** [reply]
Strange! I use network-manager-applet 1.28.0-1, which is currently the latest version in Arch.

I envy your menu ;) A hidden setting?

Are we talking about the same app?

**Reply by Lokawn** [reply]
````
$  pacman -Qs network-manager-applet
local/network-manager-applet 1.28.0-1
    Applet for managing network connections
````

Edit: I use `nm-applet --appindicator` for autostart

**Comment by tsujan** [comment, marked as answer]
Problem solved. You use `nm-applet --indicator` (https://wiki.archlinux.org/title/NetworkManager#Appindicator):

![nm-applet](https://user-images.githubusercontent.com/981076/174274469-537fc95d-28ff-4c41-aabc-68935cea6af5.png)

That's why it doesn't have a left-click menu: Appindicator uses Status Notifier, while the ordinary `nm-applet` uses the legacy tray.

Appindicator's menu is preferable because it's native (drawn by Qt).

**Comment by Lokawn** [comment]
Got it working  by prepending the auto start command for nm-applet and blueman-applet with `dbus-launch`.
But nm-applet now looks very bad (menu offset issue).
This makes the nm-applet use sys-tray.

**Reply by tsujan** [reply]
> Got it working by prepending the auto start command for nm-applet

I did exactly the opposite ;) Changed it to `nm-applet --indicator`. Why having weird GTK menus when a single Qt menu does the job?

Your report was useful to me. Thanks!
